test/e2e_env/kubernetes/gateway/gateway.go

```diff
@@ -536,6 +536,28 @@ spec:
 	})
 
 	Context("MeshLoadBalancingStrategy", func() {
+		// Introduce a shorter connection duration because, once a connection is established,
+		// the load balancer might continue routing traffic to existing instances.
+		mt := fmt.Sprintf(`
+apiVersion: kuma.io/v1alpha1
+kind: MeshTimeout
+metadata:
+  name: mt-close-connection
+  namespace: %s
+  labels:
+    kuma.io/mesh: %s
+spec:
+  targetRef:
+    kind: Mesh
+    proxyTypes:
+    - Gateway
+  to:
+  - targetRef:
+      kind: Mesh
+    default:
+      idleTimeout: 10s
```

Comment:
Is the idleTimeout actually useful?

Reply:
Might not be required in this case since we don't have long periods without any requests.